jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java

```diff
@@ -279,6 +280,7 @@ public static String decode(String encoded,String charEncoding)
      * @throws IllegalArgumentException if the input is not a valid
      *         B64 encoding.
      */
+	@SuppressWarnings("DefaultCharset")
```

Comment:
tabs vs spaces. :-)
please setup your editor to use spaces.

Reply:
This method uses the `Charset` object already in its arguments, why is errorprone flagging it?

Reply:
`charEncoding` can be null and then the platform default is used

Reply:
Does errorprone honor the `@Nullable` and `@NotNull` type annotations for situations like this?